api/project/project.go

```diff
@@ -11,16 +11,25 @@ import (
 type Project struct {
 	ID           string
 	Name         string
+	Description  string
 	Members      map[string]privilege.Level
 	DeployKeys   map[string]string
 	PadlfileHash string
 }
 
+// Summary TODO
```

Comment:
you can put the Summary struct inside the project so as to not duplicate its fields